crates/node/src/lib.rs

```diff
@@ -1234,21 +1146,15 @@ impl Node {
     }
 
     async fn call_mutate(
-        &mut self,
+        &self,
         context: &Context,
-        transaction: Transaction,
-        outcome_sender: oneshot::Sender<Result<Outcome, MutateCallError>>,
-    ) -> Result<Hash, MutateCallError> {
-        if context.id != transaction.context_id {
+        action_list: ActionMessage,
```

Comment:
as mentioned later in the call, actions won't come from the client, so this would just be the (method, args & executor ID)

Reply:
Feel free to make changes as you see fit 🙂